inst/shiny/modules/tab_nca/nca_results.R

```diff
@@ -102,9 +103,31 @@ nca_results_server <- function(id, pknca_data, res_nca, settings, grouping_vars)
         )
     })
 
+    # Provide the zip file for download
+    output$download_zip <- downloadHandler(
+      filename = function() {
+        project <- session$userData$project_name()
+        datetime <- attr(res_nca(), "provenance")$datetime
+        paste0(project, "_", format(datetime, "%d-%m-%Y"), ".zip")
+      },
+      content = function(fname) {
+        output_tmpdir <- paste0(tempdir(), "/output")
```

Comment:
nitpick: The recommended way to construct a folder path would be
```suggestion
        output_tmpdir <- file.path(tempdir(), "output")
```